akka-parsing/src/main/scala/akka/parboiled2/RuleDSLBasics.scala

```diff
@@ -92,7 +92,7 @@ trait RuleDSLBasics {
   /**
    * Matches the EOI (end-of-input) character.
    */
-  def EOI: Char = akka.parboiled2.EOI
+  final def EOI: Char = akka.parboiled2.EOI
```

Comment:
Why not `val`?

Reply:
I can remember I checked something about that, maybe I can try again.